src/utils/optimizely.js

```diff
@@ -0,0 +1,65 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
+}
+
+export const EXPERIMENT_0 = {
+  ID: 21468970242,
+  VARIANT_IDS: {
+    CONTROL: 21446530045,
+    VARIANT_1: 21458980337,
+  },
+}
+
+export const getOptimizelyProps = () => ({
+  src: process.env.ENVIRONMENT === 'production'
```

Comment:
[curious] What is the benefit of using the non-edX CDN version of the script? Also curious why we wouldn't want to keep the import in the index.html file? As is, we will need to load all the application JavaScript in the browser before we can call this function, whereas the script would load immediately in index.html.